aat/src/aat/resources/features/F-080/S-080.1.td.json

```diff
@@ -0,0 +1,35 @@
+{
+  "_guid_": "S-080.1",
+  "_extends_": "F-080_Test_Base_Data",
+  "title": "must return 200",
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid jurisdiction id",
+    "contains a valid caseType id",
+    "has the 200 OK code",
+    "contains case type details"
+  ],
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Type": "[[ANYTHING_PRESENT]]",
+      "Transfer-Encoding" : "[[ANYTHING_PRESENT]]",
+      "Strict-Transport-Security": "[[ANY_STRING_NULLABLE]]"
+    },
+    "body": {
+      "id": "FT_MasterCaseType",
+      "description": "[[ANYTHING_PRESENT]]",
+      "version": "[[ANYTHING_PRESENT]]",
+      "name" : "[[ANYTHING_PRESENT]]",
+      "jurisdiction": "[[ANYTHING_PRESENT]]",
+      "events": "[[ANYTHING_PRESENT]]",
+      "states": "[[ANYTHING_PRESENT]]",
+      "acls": "[[ANYTHING_PRESENT]]",
+      "searchAliasFields": "[[ANYTHING_PRESENT]]",
+      "case_fields": "[[ANYTHING_PRESENT]]",
+      "printable_document_url" : "[[ANY_NULLABLE]]",
+      "security_classification" : "[[ANYTHING_PRESENT]]"
```

Comment:
I know this may sound unpleasant, but we should check what actually comes inside the case type.

Reply:
Done